Doc/library/pathlib.rst

```diff
@@ -965,8 +965,8 @@ call fails (for example because the path doesn't exist):
 
 .. method:: Path.rglob(pattern)
 
-   This is like calling :meth:`Path.glob` with "``**``" added in front of the
-   given *pattern*::
+   This is like calling :func:`Path.glob` with "``**/``" added in front of the
+   given relative *pattern*:
```

Comment:
The double colon (`::`) at the end is intentional; it's ReST syntax for code block. Please leave it there.